src/pages/documentation/places/release-notes.md

```diff
@@ -0,0 +1,7 @@
+# Release Notes
+
+## January 31, 2023
+
+### Android Places 2.0.0
+
+* Major version update for [Adobe Experience Platform Location Service](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is already available as an [open source project on GitHub](https://github.com/adobe/aepsdk-places-android).
```

Comment:
already -> now

Reply:
Can we also add info on the removal of an overloaded API with this release in the notes here.